src/components/settings/settings.js

```diff
@@ -243,6 +244,25 @@ class Settings extends Component {
     return qualities;
   }
 
+  /**
+   * Prepares the badge of the quality option according to the height of its resolution
+   *
+   * @param {number} videoTrackHeight - video track
+   * @returns {Component<Badge>}
+   * @memberof Settings
+   */
+  getBadge(videoTrackHeight: number): Component<Badge> {
+    let badgeContent = '';
+    if (videoTrackHeight >= 720 && videoTrackHeight < 2160) {
+      badgeContent = 'HD';
```

Comment:
Should this be translated in other languages? If so we need to take out to localization file

Reply:
no, i asked ron